client/src/containers/PostDashboard.tsx

```diff
@@ -0,0 +1,68 @@
+import Button from "@mui/material/Button";
+import CssBaseline from "@mui/material/CssBaseline";
+import Grid from "@mui/material/Grid";
+import Box from "@mui/material/Box";
+import Container from "@mui/material/Container";
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+import Typography from "@mui/material/Typography";
+import Pagination from "@mui/material/Pagination";
+import Header from "../components/Header";
+import PostPreview from "../components/PostPreview";
+
+const theme = createTheme();
+
+export default function PostDashboard() {
+  return (
+    <ThemeProvider theme={theme}>
```

Comment:
Theme should only be at the root of the application. We shouldn't have `ThemeProviders` everywhere